src/encoded/tests/test_types_biosample.py

```diff
@@ -161,3 +161,23 @@ def test_biosample_summary_construct(testapp,
     assert res.json['object']['summary'] == (
         'Drosophila melanogaster '
         'female (10 days) liver tissue stably expressing C-terminal eGFP-tagged ATF4 under daf-2 promoter')
+
+
+def test_is_perturbed(
+    testapp,
+    biosample_1,
+    interference_genetic_modification,
+):
+    testapp.patch_json(biosample_1['@id'], {
+        'genetic_modifications': [interference_genetic_modification['@id']],
+        })
+    res = testapp.get(biosample_1['@id']+'@@index-data')
+    assert res.json['object']['perturbed'] is True
+
+
+def test_is_not_perturbed(
```

Comment:
What about a test for treatments and treatments/perturbed modifications together?

Reply:
Added 2 more tests